src/Simulation/CsharpGeneration/Resources/EntryPoint/Driver.cs

```diff
@@ -0,0 +1,206 @@
+﻿namespace @Namespace
```

Comment:
I take it this file hasn't changed, has it?

Reply:
Yes, no changes to Driver.cs besides moving things to the other files. Also, no change to Validation.cs besides renaming `Result<T>` to `Validation<T>`.